mkdocs.yml

```diff
@@ -115,6 +115,7 @@ nav:
     - CI/CD Pipeline: articles/building-ci-cd-pipeline.md
     - Authentication and Authorization: articles/authn-authz.md
     - Verifying Image Signatures: articles/verifying-signatures.md
+    - Software Provenance Workflow Using OCI Artifacts: articles/workflow.md
```

Comment:
Add this workflow.md article "link" in What's New (OCI artifacts support) - "See this article to do an end-to-end OCI artifacts workflow"

Reply:
Done